docs/REST_API-Draft/draft.md

```diff
@@ -183,16 +216,18 @@ Parametry:
 | Nazwa | Wymagany | Opis | Domyślnie |
 |-------|----------|------|-----------|
 | measurement-names | opcjonalny | nazwa poszukiwanych pomiarów | |
+| host-names | opcjonalny | nazwa hostów, z których pochodzi pomiar | |
 | time-from | opcjonalny | pomiary starsze niż podana data | |
 | time-to | opcjonalny | pomiary młodsze niż podana data | |
 | count | opcjonalny | ilosc pomiarów | 50 |
 | offset | opcjonalny | ilosc ignorowanych pierwszych wyników(do stronnicowania) | 0 |
+| only_complex | opcjonalny | tylko pomiary złożone | false |
 
 Odpowiedź:
 
 | Nazwa | Wymagany | Opis | Domyślnie |
 |-------|----------|------|-----------|
-| measurment_id | wymagany | id pomiaru |  |
+| host-name | wymagany | nazwa hosta |  |
 | measurment_name | wymagany | nazwa pomiaru |  |
```

Comment:
i z tego co mówiliśmy, to mieliśmy zamienić na measurment_id (tak pamiętam ze spotkania)